packages/util-use-case/lib/adapters/toResult.test.mjs

```diff
@@ -0,0 +1,17 @@
+import test from '@zorko-io/tool-test-harness'
+import sinon from 'sinon'
+
+import {toResult} from './toResult'
+
+const result = {}
```

Comment:
move definitions like 

```
const result = {}
const req = {}
const res = {send: sinon.stub()}
const deps = {}



```

under test(...)

or use `test.beforeEach` for initialization of test inputs